assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -192,7 +192,7 @@ function createResourceAccessChart ({ data, weekRange, gradeSelection, resourceT
 
     bar.enter()
       .append('rect')
-      .attr('tabindex', d => d.index * 10 + 1)
+      .attr('tabindex', 0)
```

Comment:
This changes see the `tabindex` strategy to use `0`, per general guidance from MDN on using `tabindex` (see Warning on https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex). The downside is that it doesn't go from link label to bar. I have asked for accessibility feedback on this, but I haven't heard back yet. See also change to old line 463.